Core/Frameworks/Baikal/Model/Config/Standard.php

```diff
@@ -54,6 +54,10 @@ class Standard extends \Baikal\Model\Config {
         "admin_passwordhash" => [
             "type"    => "string",
             "comment" => "Baïkal Web admin password hash; Set via Baïkal Web Admin",
+        ],
+        "mailbox" => [
```

Comment:
From a user standpoint, the name `mailbox` looks weird IMHO. What about `dav_auth_imap_server` instead?

Reply:
The word `mailbox` is used in sabre/dav and is the named used by PHP referencing the IMAP connection. You can see all the options PHP provides to connect to a mailbox here: https://www.php.net/manual/en/function.imap-open.php

I think the word `dav_auth_imap_server`  is only referencing the server and not all the connection options.

If you want me to change it let me know.

Reply:
I agree that the word "mailbox" is hard to understand. Without the comment, it is unclear that this is an auth backend.

Reply:
@ByteHamster I started trying to rebase this PR. But this whole `aConstants` array was removed in PR #938 

Where is this settings data supposed to go now?

Reply:
I think we can now just remove that small part from that patch since it now has been removed.

Reply:
I rebased to resolve conflicts, then added a commit to rename "mailbox"